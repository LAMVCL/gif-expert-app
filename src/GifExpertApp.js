import React from 'react'

export const GifExpertApp = () => {
    return (
        <div>
            <h2>GifExpertApp</h2>
            <hr></hr>
        </div>
    )
}
