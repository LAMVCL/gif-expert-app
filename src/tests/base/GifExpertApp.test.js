import { shallow } from "enzyme";
import { GifExpertApp } from "../../GifExpertApp";

describe('Set de pruebas para GifExpertApp', () => {
    
    test('prueba de snapshot en <GifExpertApp/> ', () => {
        
        const wrapper = shallow(<GifExpertApp/>);
        expect(wrapper).toMatchSnapshot();

    });

    test('debe de mostrar una lista de categorias ', () => {
        
        const categories = ['One punch','Tokyo Revengers'];

        const wrapper = shallow(<GifExpertApp defaultCategories={categories}/>);

        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('GifGrid').length).toBe(categories.length);
    })
    


});
