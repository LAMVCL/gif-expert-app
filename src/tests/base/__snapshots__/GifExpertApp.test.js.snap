// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Set de pruebas para GifExpertApp debe de mostrar una lista de categorias  1`] = `
<div
  className="header"
>
  <h2>
    Gif Expert App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <div
    className="contenido"
  >
    <GifGrid
      category="One punch"
      key="One punch"
    />
    <GifGrid
      category="Tokyo Revengers"
      key="Tokyo Revengers"
    />
  </div>
</div>
`;

exports[`Set de pruebas para GifExpertApp prueba de snapshot en <GifExpertApp/>  1`] = `
<div
  className="header"
>
  <h2>
    Gif Expert App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <div
    className="contenido"
  />
</div>
`;
